docs/intro.md

```diff
@@ -1,29 +1,38 @@
 ---
+title: Introduction
+hide_title: false
+draft: false
+sidebar_label: Introduction
 sidebar_position: 0
+keywords:
+  - deriv-api
+  - getting-started-deriv-api
+description: how to use deriv api, deriv api gettting started
 ---
 
-# Introduction
-
 Our Documentation is divided in several sections:
 
 - [Setting up a deriv app](/docs/application_setup.md)
+- [Terminology](category/terminology)
 - [Core Concepts](category/core-concepts)
 - [Languages](category/languages)
 - [Client Libraries](category/client-libraries)
-- [FAQ](category/faq)
 - [Guides](category/guides)
+- [FAQ](category/faq)
 
 ### Quick Start
 
-Please skim through our [Core Concept](category/core-concepts) pages, these pages will help get familiar with our `websocket server api` features, the concepts introduced in these pages are not related to any of our frameworks or libraries, so in order to use our `websocket server api` in an efficient way it's better if you grasp all the information provided there.
+# Introduction
+
+Please skim through our [Core Concept](category/core-concepts) and [Terminology](category/terminology) pages, these pages will help you get familiar with our `websocket server api` features, the concepts introduced in these pages are not related to any of our frameworks or libraries, so in order to use our `websocket server api` in an efficient way it's better if you grasp all the information provided there.
 
-When you're done with [Core Concept](category/core-concepts) you can choose your implementation approach based on the language / framework / library you want to work with in your application.
+When you're done with [Core Concept](category/core-concepts) and [Terminology](category/terminology) pages you can choose your implementation approach based on the language / framework / library you want to work with in your application.
 
 Using our [Client Libraries](category/client-libraries) is optional and you can get the same result with the tutorials provided in [Languages Section](category/languages) as well, but our client libraries provide more `robust`, `type safe` API calls for you.
 
 ### Using our websocket server with just languages features
 
-If you decided that you want to use our `websocket server` with pure languages feature of your choice please check out [Languages Section](category/languages) continue your journey from there.
+If you want to use our `websocket server` with pure languages feature of your choice please check out [Languages Section](category/languages) and continue your journey from there.
```

Comment:
If you want to use our `websocket server api` with a specific coding language, then have a look at the [Languages Section](category/languages) and continue your journey from there.